scss/_utilities.scss

```diff
@@ -461,6 +461,7 @@ $utilities: map-merge(
         lighter: $font-weight-lighter,
         normal: $font-weight-normal,
         bold: $font-weight-bold,
+        semibold: $font-weight-semibold,
```

Comment:
How about moving the new utilities to a separate PR and then rebase this one?

Reply:
Rebased everything to isolate these changes to their own commit, I think that's okay since it's clearly outlined.